src/common/components/chip-selection/ChipSelection.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import CheckIcon from '@heroicons/react/solid/CheckIcon';
+import { classNames } from '../../helpers/Tailwind.helper';
+
+interface ChipSelectionProps {
+  label: string;
+  helperText: string;
+  values: number[];
+  readonly?: boolean;
+  error?: string;
+  defaultItem: number;
+  onItemChange: (item: number) => void;
+}
+
+interface ChipProps {
+  item: number;
+  selected?: boolean;
+  readonly?: boolean;
+  onClick: (itemId: number) => void;
+}
+
+export const Chip = ({ item, selected, readonly, onClick }: ChipProps) => {
+  return (
+    <span
+      className={classNames(
+        selected ? 'font-semibold bg-green-50' : 'font-normal bg-gray-100',
+        'h-9 inline-flex items-center px-5 py-2.5 rounded-full  text-sm  text-gray-800 cursor-pointer',
+      )}
+      onClick={() => !readonly && onClick(item)}
+    >
+      {item}
```

Comment:
Selected number e bold pe design, in implementare nu.

Reply:
Font weight 600 pentru selected, 400 pentru notSelected. Este ca in design.